wiki/Tournaments/COE/COE_2025/en.md

```diff
@@ -0,0 +1,230 @@
+
+| Position | Member(s) |
+| :-- | :-- |
+| Admins | ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815), ::{ flag=DE }:: [Meyer](https://osu.ppy.sh/users/5452367), ::{ flag=DE }:: [TheHunter1](https://osu.ppy.sh/users/6496016), ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454) |
+| Mappool selector | ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815), ::{ flag=PL }:: [Bartek22830](https://osu.ppy.sh/users/6404027), ::{ flag=PL }:: [gnahus](https://osu.ppy.sh/users/12779141) |
+| Mapper | ::{ flag=PL }:: [milosz](https://osu.ppy.sh/users/13108233), ::{ flag=IT }:: [zekk](https://osu.ppy.sh/users/9704802), TBA |
+| Commentator | ::{ flag=GB }:: [SadShiba](https://osu.ppy.sh/users/10747626), TBA |
+| Referee | ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815), ::{ flag=DE }:: [TheHunter1](https://osu.ppy.sh/users/6496016), ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454) |
+
+## Links
+
+- **[Information sheet](https://docs.google.com/spreadsheets/d/1v1JULOwW6b2QXvnSnmmWLhuyIjhjiFWSTIsoSIziTpw)**
+- [COE website](https://cavoe.events)
+- [Livestream](https://twitch.tv/coevent)
+- [COE Discord server](https://discord.com/invite/d6ru6PVcSY)
+- [COE 1v1 Tournament Discord server](https://discord.gg/zJ7e6cTVFh)
+- [COE Twitter](https://twitter.com/CavoesOsuEvent)
+
+## Participants
+
+| Qualifier seed | Player |
+| :-: | :-- |
+| 1–16 | TBA |
+| 17–32 | TBA |
+| 33–48 | TBA |
+| *N/A* | TBA |
+
+## Podium
+
+This competition has not yet taken place.
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | TBA |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | TBA |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | TBA |
+
+## Mappools
+
+### Round of 16, Quarterfinals, Semifinals & Finals
+
+TBA
+
+### Qualifiers
+
+- No Mod
+  1. [Alestorm - Shipwrecked [I'LL DRINK TO THAT]](https://osu.ppy.sh/beatmapsets/1850895#osu/3802259)
+  2. [miru_maki.gjw - smile [Tournament Ver.]](https://osu.ppy.sh/beatmapsets/2399421#osu/5200725)
+  3. [XenjeS - Night Sky Solitude [Nocturne, the Eternal Nightmare]](https://osu.ppy.sh/beatmapsets/2333228#osu/5006943)
+  4. [tn-shi - Synthesis. [.Terminus ()]](https://osu.ppy.sh/beatmapsets/2088830#osu/4377326)
+- Hidden
+  1. [AAAA + Umeboshi Chazuke - Cait Sith [Cat-sith]](https://osu.ppy.sh/beatmapsets/1848926#osu/3798091)
+  2. [GOHOBI - Crazy For You [e]](https://osu.ppy.sh/beatmapsets/1900684#osu/3917923)
+- Hard Rock
+  1. [xi - FREEDOM DiVE [FOUR DIMENSIONS]](https://osu.ppy.sh/beatmapsets/39804#osu/129891)
+  2. [ESTi - HELIX (Edit ver.) [EX EX]](https://osu.ppy.sh/beatmapsets/194910#osu/462700)
+- Double Time
+  1. [Indivision & Livewire (feat. Tasha Baxter) - Won't You Stay [Forever And A Day]](https://osu.ppy.sh/beatmapsets/309744#osu/692389)
+  2. [Kurokotei - Into The Void [Expert]](https://osu.ppy.sh/beatmapsets/1112435#osu/2335757)
+
+## Ruleset
+
+### 1. General Information
```

Comment:
we use sentence case in the wiki headings https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#headings here and hereafter